index.html

```diff
@@ -5,21 +5,110 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Relaxr | Blog</title>
 
 		<!-- external CSS link -->
-		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/normalize.css">
+
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,600,700,800" rel="stylesheet">
+
 		<link rel="stylesheet" href="css/style.css">
 
 	</head>
 	<body>
 		<div id="container">
```

Comment:
I would not wrap all of my content in a container `<div>` because you can see now the header and footer do not span the full width of the page. I would have used the container '<div>' within my nav, content (wrapped around both columns) and my footer to center the content **within** those elements.